docs/Compile.rst

```diff
@@ -121,7 +156,9 @@ DF, which causes DF to use your system libstdc++ instead::
 Note that distributing binaries compiled with newer GCC versions requires end-
 users to delete libstdc++ themselves and have a libstdc++ on their system from
 the same GCC version or newer. For this reason, distributing anything compiled
-with GCC versions newer than 4.8 is discouraged.
```

Comment:
The first version we had issues with was 4.9, since Arch adopted it before Debian, so the 4.8 was intentional because it should work on pretty much any reasonably up-to-date system. However, if you have to install a different version of gcc for compatibility, an older version (>=4.5) would probably be better, although support for 4.5-4.7 (maybe 4.8 too) ought to be equally widely available on different distros.